{"uuid": "2b910523-11ba-4db6-880b-9e6737b5c34d", "children": ["a31e29cd-60f3-4ded-843e-7296a713a738"], "befores": [{"name": "login", "status": "broken", "statusDetails": {"message": "json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n", "trace": "  File \"D:\\Python3\\Proj\\venv\\lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\Python3\\Proj\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\Python3\\Proj\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 932, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"D:\\Python3\\Proj\\testcases\\conftest.py\", line 23, in login\n    if str(assert_str) == str(json.loads(rep)[\"code\"]):\n  File \"D:\\Python3\\lib\\json\\__init__.py\", line 357, in loads\n    return _default_decoder.decode(s)\n  File \"D:\\Python3\\lib\\json\\decoder.py\", line 337, in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n  File \"D:\\Python3\\lib\\json\\decoder.py\", line 355, in raw_decode\n    raise JSONDecodeError(\"Expecting value\", s, err.value) from None\n"}, "start": 1647249087901, "stop": 1647249087913}], "start": 1647249087901, "stop": 1647249087945}